app/services/schools/generate_chart_configuration.rb

```diff
@@ -0,0 +1,58 @@
+require 'dashboard'
+
+module Schools
+  class GenerateChartConfiguration
+    def initialize(
+        school,
+        aggregated_meter_collection,
+        fuel_types_for_analysis = school.fuel_types_for_analysis,
+        dashboard_fuel_types = DashboardConfiguration::DASHBOARD_FUEL_TYPES,
+        dashboard_page_configuration = DashboardConfiguration::DASHBOARD_PAGE_GROUPS
+      )
+
+      @school = school
+      @aggregated_meter_collection = aggregated_meter_collection
+      @fuel_types_for_analysis = fuel_types_for_analysis
+      @dashboard_fuel_types = dashboard_fuel_types
+      @dashboard_page_configuration = dashboard_page_configuration
+    end
+
+    def generate
+      Rails.logger.info "Generating chart configuration for #{@school.name} - using default values"
+      page_and_chart_config = {}
+      pages.each do |page|
+        page_and_chart_config[page.to_sym] = page_config(page)
+      end
+      @school.configuration.update(dashboard_charts: page_and_chart_config)
+    end
+
+  private
+
```

Comment:
This is in here temporarily for now, doesn't get called will be set up properly once we have some configurations created.